Gemfile

```diff
@@ -21,6 +21,7 @@ gem 'jbuilder', '~> 2.7'
 # gem 'redis', '~> 4.0'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
+gem 'pagy'
```

Comment:
you started to add new gems in the bottom, so add all new in the bottom of the file